.github/workflows/publish-npm-packages.yml

```diff
@@ -48,11 +48,56 @@ jobs:
         run: |
           EXTENSIONS=$(node ${{ env.SCRIPTS_DIR }}/get-extensions.js)
           echo "::set-output name=extensions::{\"include\":${EXTENSIONS}}"
-  publish:
+  publish-core:
     if: github.repository == 'ampproject/amphtml'
     environment: bento_publishing
     needs: setup
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.events.inputs.amp-version }}
+      - uses: actions/setup-node@v2
+        with:
+          registry-url: https://registry.npmjs.org
+          node-version: 16
```

Comment:
nit: use `lts/*`

Reply:
should I change it for extension `publish` job as well?